packages/frontend/src/components/sign/SignTransferMultipleAccounts.js

```diff
@@ -0,0 +1,127 @@
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import ArrowUpImage from '../../images/icon-arrow-up-green.svg';
+import DangerIcon from '../../images/icon-danger-circle.svg';
+import FormButton from '../common/FormButton';
+import Container from '../common/styled/Container.css';
+import SafeTranslate from '../SafeTranslate';
+
+const CustomContainer = styled(Container)`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: #25282a;
+    text-align: center;
+
+    .title {
+        margin-top: 23px;
+
+        h2 {
+            font-weight: 900;
+            font-size: 22px;
+            color: #24272a;
+        }
+    }
+
+    && .text {
+        color: #72727a;
+        margin-top: 24px;
+    }
+
+    .buttons {
+        display: flex;
+        width: 100%;
+        margin-top: 40px;
+
+        button {
+            flex: 1;
+
+            &:last-of-type {
+                margin-left: 30px;
+
+                @media (min-width: 768px) {
+                    margin-left: 50px;
+                }
+            }
+        }
+    }
+
+    .fees {
+        width: 100%;
+        border: 1px solid #f0f0f1;
+        padding: 15px;
+        border-radius: 8px;
+        margin-top: 30px;
+        color: #72727a;
+
+        b {
+            color: #25282a;
+        }
+
+        .fees-line {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            margin-top: 12px;
+
+            .tgas {
+                color: #00c08b;
+                position: relative;
+
+                :after {
+                    content: "";
+                    position: absolute;
+                    background: url(${ArrowUpImage}) center top no-repeat;
+                    width: 16px;
+                    height: 17px;
+                    left: -22px;
+                    top: 1px;
+                }
+            }
+        }
+    }
+`;
+
+const SignTransferMultipleAccounts = ({
+    handleCancel,
+    signCallbackUrl,
+    submittingTransaction,
+    signTransactionSignerId,
+}) => (
+    <CustomContainer className="small-centered">
+        <div className="icon">
+            <img src={DangerIcon} alt="Retry" />
+        </div>
+        <div className="title">
+            <h2>
+                <Translate id="sign.multipleAccountsError.title" />
+            </h2>
+        </div>
+        <div className="text">
+            <SafeTranslate
+                id="sign.multipleAccountsError.body"
+                data={{ signCallbackUrl, signTransactionSignerId }}
+            />
+        </div>
+        <div className="buttons">
+            <FormButton
+                onClick={handleCancel}
+                disabled={submittingTransaction}
+                color="gray-blue"
+            >
+                <Translate id="button.cancel" />
+            </FormButton>
+            <FormButton
+                linkTo={signCallbackUrl}
```

Comment:
The `linkTo` automatically opens a new tab instead of redirect, but might be better to just fire a `onClickReturnToApp` function `onClick` like the [`LoginWrapper`](https://github.com/near/near-wallet/blob/master/packages/frontend/src/routes/LoginWrapper.js#L48) does?

Reply:
Thanks! Updated 👍